src/view/configuration/streamingMediaSection.jsx

```diff
@@ -12,38 +12,58 @@ governing permissions and limitations under the License.
 
 import React from "react";
 import PropTypes from "prop-types";
-import { number, object, string } from "yup";
+import { number, object, string, lazy } from "yup";
 import { useField } from "formik";
 import SectionHeader from "../components/sectionHeader";
 import FormElementContainer from "../components/formElementContainer";
 import FormikTextField from "../components/formikReactSpectrum3/formikTextField";
 import FormikNumberField from "../components/formikReactSpectrum3/formikNumberField";
 import isNonEmptyString from "../utils/isNonEmptyString";
 
-const getDefaultSettings = () => {
-  return {
-    channel: "",
-    playerName: "",
-    appVersion: "",
-    adPingInterval: 10,
-    mainPingInterval: 10,
-  };
-};
+const getDefaultSettings = () => ({
+  channel: "",
+  playerName: "",
+  appVersion: "",
+  adPingInterval: 10,
+  mainPingInterval: 10,
+});
+
 export const bridge = {
   getInstanceDefaults: () => ({
     streamingMedia: getDefaultSettings(),
   }),
+
   getInitialInstanceValues: ({ instanceSettings }) => {
     if (!instanceSettings.streamingMedia) {
-      instanceSettings.streamingMedia = getDefaultSettings();
+      return bridge.getInstanceDefaults();
```

Comment:
The extension configuration was crashing because the instanceSettings was returned. The instance settings was containing keys that could potentially override other bridge method from different sections. We need to return an object that contains just the `streamingMedia` key.